modules/Ui.js

```diff
@@ -0,0 +1,29 @@
+import Store from './Store';
+
+export default class Ui {
+  static addLists = (list) => {
+    const toDoList = document.querySelector('.to-do-list');
+    const newItem = document.createElement('li');
+
+    newItem.setAttribute('class', 'add-to-list');
+    newItem.setAttribute('id', list.index);
+    newItem.setAttribute('data-index', list.index);
+    newItem.innerHTML = `
+      <div data-index="${list.index}" class="checkbox">
+
+        <input data-index="${list.index}" type="checkbox" ${list.completed ? 'checked' : ''} class="checkbox-input">
+        <label class="checkbox-label ${list.completed ? 'strikethrough-completed' : ''}" data-index="${list.index}">${list.description}</label>
+        
+      </div>
+      <i data-index="${list.index}" class="fa-solid fa-trash-can remove-btn"></i>
+      `;
```

Comment:
- `[OPTIONAL]`: Please try to reduce the usage of the `innerHTML` method in your day-to-day code. because `innerHTML` have some serious security issue like `SQL-Injection`. 
- Please try to use `insertAjacentHtml` instead of `innerHTML`.